armadillo/src/main/java/org/molgenis/armadillo/controller/StorageController.java

```diff
@@ -116,14 +117,58 @@ public void uploadObject(
         Map.of(PROJECT, project, OBJECT, object));
   }
 
-  private void addObject(String project, String object, MultipartFile file) {
+  @Operation(summary = "Upload a csv file to a project")
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "204", description = "Object uploaded successfully"),
+        @ApiResponse(responseCode = "404", description = "Unknown project"),
+        @ApiResponse(responseCode = "409", description = "Object already exists"),
+        @ApiResponse(responseCode = "401", description = "Unauthorized")
+      })
+  @PostMapping(
+      value = "/projects/{project}/objects/csv",
```

Comment:
we will not have problems with other meanings of this path? often the path is GET {project}/objects/{id} and thus also posts? I conflicts are an issue the  {project}/csv/{id} might bet better. Please advice?

Reply:
I based the endpoint on the objects endpoint (which uploads any file, not just csv), but maybe `/projects/{project}/csv` would be better. I didn't add in the id in the URL for consistency (because the `/objects` endpoint doens't have it in there).